lib/src/main/java/com/orange/ods/compose/component/list/OdsListItem.kt

```diff
@@ -311,6 +341,100 @@ fun Modifier.iconType(iconType: OdsListItemIconType): Modifier {
     })
 }
 
+/**
+ * A modifier that allows to configure the icon type in an [OdsListItem].
+ */
+private interface OdsListItemIconTypeModifier : Modifier.Element {
+    val iconType: OdsListItemIconType
+}
+
+//endregion
+
+//region OdsListItem Trailing
+
+private fun Modifier.trailingModifier(trailing: OdsListItemTrailing): Modifier = when (trailing) {
+    is OdsCheckboxTrailing -> toggleable(
```

Comment:
We should use the `then` method, otherwise everything in `this` will simply be replaced by `togglable` instead of being concatenated:

```kotlin
is OdsCheckboxTrailing -> then(toggleable(
    value = trailing.checked.value,
    role = Role.Checkbox,
    enabled = trailing.enabled,
    onValueChange = { trailing.onCheckedChange() }
))
```

Same comment for `OdsRadioButtonTrailing` and `OdsSwitchTrailing` cases.